api/WIP/flow-management.yaml

```diff
@@ -0,0 +1,1050 @@
+          tokenUrl: https://localhost:8090/oauth2/token
+          scopes:
+            system: Access to system management APIs
+
+  schemas:
+    FlowListResponse:
+      type: object
+      properties:
+        totalResults:
+          type: integer
+          description: "Number of results that match the listing operation."
+          example: 5
+        startIndex:
+          type: integer
+          description: "Index of the first element of the page, which will be equal to offset + 1."
+          example: 1
+        count:
+          type: integer
+          description: "Number of elements in the returned page."
+          example: 3
+        flows:
+          type: array
+          items:
+            $ref: '#/components/schemas/BasicFlowDefinitionResponse'
+        links:
+          type: array
+          items:
+            $ref: '#/components/schemas/Link'
+          description: "Navigation links for pagination."
+      example:
+        totalResults: 5
+        startIndex: 1
+        count: 3
+        flows:
+          - id: "641af221-75ff-4247-ab1f-ff139040a91b"
+            flowType: "AUTHENTICATION"
+            name: "Basic Authentication Flow"
+            version: 3
+            createdAt: "2024-01-15T10:20:30Z"
+            updatedAt: "2024-02-20T14:25:35Z"
+          - id: "2sd4f6g7-h8i9-0j1k-2l3m-4n5o6p7q8r9s"
+            flowType: "REGISTRATION"
+            name: "Basic Registration Flow"
+            version: 7
+            createdAt: "2024-03-10T09:15:25Z"
+            updatedAt: "2024-04-12T11:30:40Z"
+          - id: "b12c3d45-6789-4e0f-9abc-def123456789"
+            flowType: "REGISTRATION"
+            name: "Google Registration Flow"
+            version: 1
+            createdAt: "2024-05-05T08:10:20Z"
+            updatedAt: "2024-06-01T12:00:00Z"
+        links:
+          - href: "/flows?offset=3&limit=3"
+            rel: "next"
+
+    FlowVersionListResponse:
+      type: object
+      properties:
+        totalVersions:
+          type: integer
+          description: Total number of versions available for this flow
+          example: 8
+        versions:
+          type: array
+          items:
+            $ref: '#/components/schemas/FlowVersionInfo'
+          description: List of flow versions (most recent first)
+      example:
+        totalVersions: 8
+        versions:
+          - version: 8
+            createdAt: "2024-12-01T15:30:00Z"
+            updatedAt: "2024-12-01T15:30:00Z"
+            isLatest: true
+          - version: 7
+            createdAt: "2024-11-25T10:15:00Z"
+            updatedAt: "2024-11-25T10:15:00Z"
+            isLatest: false
+          - version: 6
+            createdAt: "2024-11-20T14:20:00Z"
+            updatedAt: "2024-11-20T14:20:00Z"
+            isLatest: false
+
+    FlowVersionInfo:
+      type: object
+      required:
+        - version
+        - createdAt
+        - updatedAt
+        - isLatest
```

Comment:
Does this property should be `isActive` ?

I hope this is to indicate which is the version is being currently used.